### DISCUSSION TITLE
Cancel uploading/downloading

### DISCUSSION THREAD

**Discussion by ntnguyen1234** [9001/copyparty#77] [Q&A] [answered]
Hi. Thanks for the project. I wonder if there is any ways users can cancel while downloading/uploading some files?

**Comment by 9001** [comment, marked as answer]
hey 👋 thx for trying copyparty!

there is currently no way to cancel an upload; this would be a good feature to add. I think I will combine it with the unpost feature, so the files listed in the 🧯 tab will also include unfinished uploads.

as for downloads, I'm not sure I understand - this capability should already be provided by the browser, right? At least in the case of chrome and Firefox, there is an ❎ button next to the active downloads to cancel them. So I'm not sure copyparty should try to do anything about this. But please correct me if I'm wrong :>

**Reply by ntnguyen1234** [reply]
Ah, you're right. I forgot that it's easier to handle downloading with browser than the site.

Thank you. I'll look forward to upload cancel feature 🙂
